Automattic/wp-calypso
# Run PHPCS in CI for FSE plugin

#### Changes proposed in this Pull Request
* Run phpcs lint check on changed files in the FSE plugin
* Expand FSE plugin GH workflow to run on changes to yarn lock file and to apps/full-site-editing

#### Testing instructions
* Lint check should work and should fail on errors

I verified that this does the following:
1. Fails when a change to a PHP file is not formatted correctly
2. Does not try to lint non-php files
3. Runs on changes to yarn.lock
